src/components/JsonSchemaViewer.tsx

```diff
@@ -160,7 +179,15 @@ const JsonSchemaViewerInner = ({
       onMouseLeave={onMouseLeave}
       style={{ maxHeight }}
     >
-      <PathCrumbs parentCrumbs={parentCrumbs} />
+      {showExpandAll ? (
+        <HStack w="full" fontFamily="mono" fontSize="sm" lineHeight="none" bg="canvas-pure" px="px" color="light">
+          <Box flex={1}>&nbsp;</Box>
+          <Button pl={1} mr={1} size="sm" appearance="minimal" onClick={onCollapseExpandAll}>
```

Comment:
I would like to only show this button when the schema as any properties with schema definition that can be expanded/collapsed. 

How can I detect this?